unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -60,6 +61,13 @@ public class BackpackEditorHUDController
             wearableGridController.OnWearableEquipped += EquipWearable;
             wearableGridController.OnWearableUnequipped += UnEquipWearable;
 
+            avatarSlotsHUDController.OnToggleSlot += ToggleSlot;
+
+            ColorPresetsSO colorPresets = Resources.Load<ColorPresetsSO>("BackpackColorPickerPresets");
```

Comment:
I think I would prefer loading this in the plugin and then injecting it

Reply:
Also, instead of the resource load let's have a serializable field and drag it from the project so we can move it out of the resources folder

Reply:
I was able to move it to the view and make it serializable 👍 Thanks!